docs/IoT.js-API-GPIO.md

```diff
@@ -124,30 +91,18 @@ Emitted after GPIO device is successfully initialized.
 
 Emitted after GPIO device is successfully released.
 
-#### `'setPin(pin: Number, direction: String, mode:String)'`
+#### `'open(pin: Number, direction: String, mode:String)'`
```

Comment:
Please explain parameters type or role following lines as above. The form is not unified.